pkg/tftp/tftp.go

```diff
@@ -319,19 +320,27 @@ func executeGet(client ClientIf, host, port string, files []string) error {
 			}
 		}
 
-		nR := 0
 		data := make([]byte, 0)
 		for {
-			fmt.Println("loop")
-			readData := make([]byte, 10)
-			rD, err := resp.Read(data)
+			// Transmission with less than 512 byte indicates termination of connection
+			// See: https://datatracker.ietf.org/doc/html/rfc1350#section-2
+			readData := make([]byte, 512)
```

Comment:
When testing this I realized, that the padding for binary sized might not work correctly.

I created a file using random bytes 
```
sudo dd if=/dev/random of=513 bs=1 count=513
```
which is just slightly bigger than the block size.

Then I retrieved the file using the `get` command. I checked the size and realized, that it was a few bytes larger than it should be. Diffing the files using `xxd` proved that there were \x00 bytes that should not be there:

```
$ diff <(xxd 513) <(xxd /tmp/513)
32,33c32,33
< 000001f0: 96aa e1d9 e28b 4515 d0d0 21b3 da82 160e  ......E...!.....
< 00000200: ec                                       .
---
> 000001f0: 96aa e1d9 e28b 4500 0000 0000 0000 0000  ......E.........
> 00000200: 15d0 d021 b3da 8216 0eec  
```

Reply:
iMpRoVeD tHe cOdE :D